src/getConfig.js

```diff
@@ -28,7 +28,8 @@ const defaultConfig = {
   linters: {},
   ignore: [],
   subTaskConcurrency: 1,
-  renderer: 'update'
+  renderer: 'update',
+  verboseMode: false
```

Comment:
Can't it be just `verbose`?

Reply:
If you are using `verboseMode` because of `debugMode`, you need not. `debugMode` was required because `debug` was used for the `debug` library.